sdk/cosmos/azure-cosmos-benchmark/src/test/java/com/azure/cosmos/benchmark/ReadMyWritesConsistencyTest.java

```diff
@@ -168,7 +168,7 @@ DocumentCollection getCollectionDefinitionWithRangeRangeIndex() {
         List<IncludedPath> includedPaths = new ArrayList<>();
         IncludedPath includedPath = new IncludedPath();
         includedPath.setPath("/*");
-        Collection<Index> indexes = new ArrayList<>();
+        List<Index> indexes = new ArrayList<>();
         Index stringIndex = Index.range(DataType.STRING);
         BridgeInternal.setProperty(ModelBridgeInternal.getJsonSerializable(stringIndex), "precision", -1);
```

Comment:
can this be updated to use the public apis for setting precision.

Reply:
Sure can, has changed across all the places.